airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -383,43 +382,10 @@ public MediaLibraryStatistics getStatistics() {
         return statistics;
     }
 
-    /**
-     * Deletes old versions of the index file.
-     */
-    private void deleteOldIndexFiles() {
-        for (int i = 2; i < INDEX_VERSION; i++) {
-            File file = getIndexFile(i);
-            try {
-                if (FileUtil.exists(file)) {
-                    if (file.delete()) {
-                        LOG.info("Deleted old index file: " + file.getPath());
-                    }
-                }
-            } catch (Exception x) {
-                LOG.warn("Failed to delete old index file: " + file.getPath(), x);
-            }
-        }
-    }
-
-    /**
-     * Returns the index file for the given index version.
-     *
-     * @param version The index version.
-     * @return The index file for the given index version.
-     */
-    private File getIndexFile(int version) {
-        File home = SettingsService.getAirsonicHome();
-        return new File(home, "airsonic" + version + ".index");
-    }
-
```

Comment:
In the previous review, file operations related to Index were encapsulated in IndexManager.
This also moves to IndexManager.